### DISCUSSION TITLE
Custom scenes bound to F# buttons

### DISCUSSION THREAD

**Discussion by Hintzy** [kinsi55/CS_BeatSaber_Camera2#118] [Q&A] [answered]
Question about binding custom scenes to F1, F2, F3, etc.   It seems like F2 and F4 have functions bound to them by default.  F2 seems to toggle my blocks between visible/hidden as well as pause the song on the initial press.  And when I press F4 it restarts the song that I'm playing.  I've looked through the BSInstances subdirectories to try to find a file that has key bindings with no success.  Various queries via google haven't yielded any information on this either, and I didn't see anything about it mentioned in Fefeland's youtube tutorial, or in the Camera2 wiki. The work around I've employed so far is just to not use these F2 or F4 for custom scenes, and instead bind my custom scenes to F1, F3, F5, and so on, but I thought I'd drop an inquiry here to see if you have any insight on this.  Thank you!

**Comment by kinsi55** [comment, marked as answer]
I think its songcore that binds those, either that or sirautil. Check their configs, there's a setting to disable those

**Reply by Hintzy** [reply]
You nailed it.  It was SiraUtil.  Here's a screenshot of the config file w/ directory with my values changes to F11 & F12.  I had issues when trying to assign the keys to empty quotes (""). 

![SiraUtil config](https://github.com/user-attachments/assets/19963f27-27f7-4bc9-a8bf-9cdf94cb7e39)
